client/client_test.go

```diff
@@ -42,6 +43,24 @@ func TestClient_Cred_PathMissing(t *testing.T) {
 	}
 }
 
+func TestClient_Cred_EnvOverride_ExplicitPath(t *testing.T) {
```

Comment:
nit: I would name the 2 test cases "ExplicitConfig" and "EmptyConfig" to refer to the test setup - the old test on line 39 should be renamed to TestClient_Cred_BinaryPathMissing to correctly capture the intent of the test.

Reply:
Okay, that makes more sense